### DISCUSSION TITLE
I win a block and the reward do not arrive

### DISCUSSION THREAD

**Discussion by ummetsunger** [Chia-Network/chia-blockchain#8874] [Support]
photo of the problem: https://ibb.co/3N4RrRB

Greeting,

I won a block at a space pool but didn’t get a prize on my wallet.

I won blok #979189 in space pool and there is no revenue yet
10.10.2021 20:47

Hash is : f0698d244f3752a57a29593084360ba7821a4ac16172bb766d86beec64e925c7

My farmer's address for the prize is the one that did not receive the prize from the block won :
xch1z79435kflzl6lrv3ywglhvypse3eqrhqk7trmd7cu220spcaqd0s982mrn

The farmer's address to which I receive rewards from the space pool is :
0x446046ac16ad1bbc7c616aa3c25c3fb8c45ae6e3f2f8a836fba26dd310df1eb1

Please help me with this problem, thanks a lot in advance.

I will send you in addition what it takes from the data to solve this problem.

Best regards,
Dejan

**Comment by plot14** [comment]
have the same problem  can i have assistance ?
